test/e2e/index.ts

```diff
@@ -296,7 +299,7 @@ test('can inject agents', async (t) => {
             installed_app_id, agent_key: agent_key_1, dnas: [{hash, nick}]
         })
         t.ok(result)
-        const app1_cell  = result.slots['thedna'].base_cell_id
+        const app1_cell  = result.cell_data[0].cell_id
```

Comment:
This is a big improvement. I was finding prior to this that you would necessarily have to call `registerDna` prior to calling installApp because `base_cell_id` wouldn't give you the NEW cell_id but, as stated, the cell_id of the "base", which is without the properties and uid that you meant to (and did) install it with. Filling out cell_data with this instead is way better. thanks.